pkg/apis/jenkins.io/v1/types_users.go

```diff
@@ -11,6 +11,13 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +k8s:openapi-gen=true
 
```

Comment:
nit: You can define a type for this.

```Go
type UserType string 

const (
    UserTypeLocal UserType = "User"
)
```

Reply:
Good pointer, thanks.
I tried this out, but it didn't really seem to help much as it is ultimately is compared against the Kind in the Subject, which is itself a String. I found the _string_ constants were already defined there, though, so switched to using those.

Reply:
@ccojocar, are you happy with this now (i.e. kept as `string`, but using the client's already defined constants).

Reply:
It's fine.